cmake/onnxruntime_common.cmake

```diff
@@ -52,6 +52,10 @@ source_group(TREE ${REPO_ROOT} FILES ${onnxruntime_common_src})
 
 add_library(onnxruntime_common ${onnxruntime_common_src})
 
+if (use_telemetry)
```

Comment:
Do you declare `use_telemetry` anywhere? Use [`option`](https://cmake.org/cmake/help/latest/command/option.html) so this option shows up for the user to choose when you configure CMake (else the user has to guess/look at the source code to know this option exists)

Reply:
Here's an example on how to do it: https://github.com/microsoft/onnxruntime/blob/fef721c4f24d137a509786058296c2981686ad51/samples/c_cxx/CMakeLists.txt#L16

Reply:
And onnxruntime was prefixing their options with `onnxruntime_` so it doesn't get mixed up with other CMake options/other projects in the same workspace. We should either use it or start using a `winml_` prefix for our options.